kitsune/users/templatetags/jinja_helpers.py

```diff
@@ -84,16 +84,15 @@ def public_email(email):
 
 def unicode_to_html(text):
     """Turns all unicode into html entities, e.g. &#69; -> E."""
-    return "".join([u"&#%s;" % ord(i) for i in text])
+    return ''.join(['&#%s;' % ord(i) for i in text])
 
 
 @library.global_function
 def user_list(users):
     """Turn a list of users into a list of links to their profiles."""
-    link = u'<a class="user secondary-color" href="%s">%s</a>'
-    list = u", ".join(
-        [link % (escape(profile_url(u)), escape(display_name(u))) for u in users]
-    )
+    link = '<a class="user" href="%s">%s</a>'
```

Comment:
Did we mean to lose `secondary-color` here?

Reply:
nope